### PR TITLE
LPS-199416 Add POSHI automation | Master

### PR DESCRIPTION
**Pull request by alessandromalves**
This is only for master
https://liferay.atlassian.net/browse/LPS-199416

Hi @Tim-Cao, on the [original bug ticket](https://liferay.atlassian.net/browse/LPS-198068), in step 9 it was said that it was necessary to access in a different browser to notice what was happening. I generated a bundle with the changes reverted and saw that the same thing happened if you logged out with admin and visited the page as a guest in the same browser. So this step was not followed. 

cc.: @david-gutierrez-mesa

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 1]
```diff
@@ -1602,6 +1602,56 @@ definition {
...
+
+			Staging.openStagingAdmin(siteURLKey = "${randomSiteURLKey}-staging");
+
+			Staging.publishCustomPublication();
+		}
+
+		task ("When this page is visited by a guest user") {
+			User.logoutPG();
+
+			Navigator.openSpecificURL(url = "http://localhost:8080/web/${randomSiteURLKey}/test-content-page-name");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1644]:
```
			ContentPagesNavigator.openViewContentPage(
				pageName = "Test Content Page Name",
				siteName = ${randomSiteName});

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 2]
```diff
@@ -1602,6 +1602,56 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-198068. View documents and media URL of Image Fragment is correctly displayed on the Live environment."
+	@priority = 4
+	test ViewDocumentsAndMediaURLOfImageFragmentIsCorrectlyDisplayedOnLive {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1607]: ViewDisplayedImageOnLiveSiteAsGuest

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 3]
```diff
@@ -1602,6 +1602,56 @@ definition {
...
+		task ("Given a content page with a image fragment and published to live") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1608]: Given a user displays an Image on content page in live site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase [hunk 4]
```diff
@@ -1602,6 +1602,56 @@ definition {
...
+			Staging.publishCustomPublication();
+		}
+
+		task ("When this page is visited by a guest user") {
+			User.logoutPG();
+
+			Navigator.openSpecificURL(url = "http://localhost:8080/web/${randomSiteURLKey}/test-content-page-name");
+		}
+
+		task ("Then the user will also be able to view the fragment image") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithStaging.testcase:1647]: Then the guest could view the displayed image

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @alessandromalves, could you modify according to comments? Thanks

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**